mimesis/providers/numbers.py

```diff
@@ -42,8 +43,22 @@ def integers(self, start: int = 0, end: int = 10,
         :Example:
             [-20, -19, -18, -17]
         """
-        return self.random.randints(
-            length, start, end)
+        return self.random.randints(length, start, end)
+
+    def matrix(self, m: int = 3, n: int = 3,
+               start: int = 0, end: int = 10) -> list:
+        """Creates a m x n matrix of random integers.
+
+        :param m: Number of rows.
+        :param n: Number of columns.
+        :param start: Start range.
+        :param end: End range.
+        :return: Matrix with m rows and n columns.
+
+        :Example:
```

Comment:
Don't you think that it will be much useful if we add support for the variety of types ( float, integer, complex etc.)?

Reply:
Yes, I was thinking the same thing. We should add an argument where we can specify the returned type.

Before implementing this feature, I would like to make some changes to this provider, please see #769